### PR TITLE
fix: sitemap url

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
- Remove ``www`` sub domain from the url in sitemap
- Add the version directory in the additional page location in the site map see https://github.com/ansys/pymapdl/blob/543819dd29d6e15e2c67c477eebf4e664c4d0e32/sitemap.xml#L16

### CODE REVIEW

File: _doc-gen-sitemap/action.yml [hunk 1]
```diff
@@ -58,8 +58,12 @@ runs:
 
         # Add other URLs
         while IFS= read -r -d '' file; do
-          url=${file#$input_dir}
-          url=${url%.html}
+          if [[ $file == *"$input_dir"* ]]; then
+            url=${file%.html}
+          else
+            # $input_dir is not in the file path, add it to the begining of the URL
```
Comment (MaxJPRey) [inline _doc-gen-sitemap/action.yml:64]:
```suggestion
            # $input_dir is not in the file path, add it to the beginning of the URL
```

**Review by RobPasMue — APPROVED** [review]

**Review by jorgepiloto — APPROVED** [review]

File: _doc-gen-sitemap/action.yml [hunk 2]
```diff
@@ -40,7 +40,7 @@ runs:
       shell: bash
       run: |
         # Declare the website name and the output file for the sitemap
-        website="https://www.${{ inputs.cname }}"
+        website="https://${{ inputs.cname }}"
```
[the remark this answers is not in the document]
Comment (RobPasMue) [reply _doc-gen-sitemap/action.yml:43]: Agreed - but that should be done everywhere. We can fix it in a full actions review, don't you agree?

[the remark this answers is not in the document]
Comment (Revathyvenugopal162) [reply _doc-gen-sitemap/action.yml:43]: yes, in that case if you agree, we can merge this PR and fix it in a full actions review, especially related to the doc actions

[the remark this answers is not in the document]
Comment (RobPasMue) [reply _doc-gen-sitemap/action.yml:43]: Please proceed to merge it and track this request with a new issue

**Pull request merged**